sphinx_gallery/tests/testconfs/src/plot_1.py

```diff
@@ -1,7 +1,9 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
 """
```

Comment:
no no - it's not encoding lines, it's sphinx references. People put stuff like

```
.. _my_ref:

My title
--------
```

which you're technically not needed to do since SG auto generates references, but [people do it anyway](https://github.com/mne-tools/mne-python/blob/master/examples/io/plot_read_events.py) :-)